tests/testthat/test-derive_param_tte.R

```diff
@@ -928,3 +928,208 @@ test_that("list_tte_source_objects Test 14: expected objects produced", {
 
   expect_dfs_equal(expected_output, observed_output, keys = c("object"))
 })
+
+# Test 15: "derive_param_tte detects duplicates when check_type = 'warning'`
+test_that("derive_param_tte detects duplicates when check_type = 'warning'", {
```

Comment:
Please format as described in [Set up the Test Script](https://pharmaverse.github.io/admiraldev/dev/articles/unit_test_guidance.html#set-up-the-test-script).
You can do this easily with the addin from pharmaverse4devs (see [Addin `pharmaverse4devs::format_test_that_file()`](https://pharmaverse.github.io/admiraldev/dev/articles/unit_test_guidance.html#addin-pharmaverse4devsformat_test_that_file))

Reply:
Done. Double checking some changes and pushing